apps/angular-example-counter/src/app/store/store.service.ts

```diff
@@ -1,21 +1,22 @@
 import { Injectable, OnDestroy } from '@angular/core';
-import { RxStoreEffect } from '@rx-store/rx-store';
+import { Effect, spawnRootEffect } from '@rx-store/core';
+import { Observable, Subscription } from 'rxjs';
+import { RootAppStore } from '../types';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class StoreService implements OnDestroy {
```

Comment:
Any reason not to rename to `ManagerService`? That would avoid the `store.store` stuff, right? At least it'd be `manager.store` which is a bit less kludgy..

Reply:
addressed in f0d592a by calling it StoreManagerService